File: sechi/Objects/SEPushNoAnimationSegue.m
//
//  SEPushNoAnimationSegue.m
//  sechi
//
//  Created by karolszafranski on 08.05.2014.
//  Copyright (c) 2014 TopCoder Inc. All rights reserved.
//

#import "SEPushNoAnimationSegue.h"

@implementation SEPushNoAnimationSegue

- (void) perform {
    [[self.sourceViewController navigationController] pushViewController:self.destinationViewController animated:NO];
}

@end

File: sechi/Objects/SEPushNoAnimationSegue.swift
//
//  SEPushNoAnimationSegue.swift
//  sechi
//
//  Created by TCSASSEMBLER on 2014-06-13.
//  Copyright (c) 2014 TopCoder. All rights reserved.
//

/**
 *  Move to the next screen without an animation.
 */
class SEPushNoAnimationSegue: UIStoryboardSegue {

    override func perform() {
        self.sourceViewController.navigationController.pushViewController(self.destinationViewController, animated: false)
    }

}
